lighthouse-core/test/report/report-generator-test.js

```diff
@@ -104,7 +104,7 @@ describe('ReportGenerator', () => {
       expect(lines.length).toBeGreaterThan(100);
       expect(lines.slice(0, 2).join('\n')).toMatchInlineSnapshot(`
```

Comment:
let's use the first 3 lines now to test both category and audit lines?

Reply:
Yes sure